pkg/api/norman/customization/cluster/validator.go

```diff
@@ -805,3 +746,30 @@ func validateGKEPrivateClusterConfig(spec *v32.ClusterSpec) error {
 	}
 	return nil
 }
+
+// isNewerVersion returns true if updated versions semver is newer and false if its
+// semver is older. If semver is equal then metadata is alphanumerically compared.
+func isNewerVersion(prevVersion, updatedVersion string) (bool, error) {
```

Comment:
This function does not appear to be used.  And appears to be defined again in pkg/image/external/external.go